### ISSUE TITLE
File Path leads to a 'Syntax error, unrecognized expression: output/bib/resources.bib'

### ISSUE THREAD

**Issue by TuningGuide** [vkaravir/bib-publication-list#14]
I would change it to something like:
```
return function(bibSrc, bibElemId, opts) {
...
	    var $bibSrc = null;
	    if(bibSrc.indexOf('/') === -1) {
		    $bibSrc = $(bibSrc);
	    }

        if ($bibSrc && $bibSrc.length) { // we found an element, use its HTML as bibtex
            new Bib2HTML($bibSrc.html(), $pubTable, options);
            $bibSrc.hide();
        } else { // otherwise we assume it is a URL
            var callbackHandler = function(data) {
                new Bib2HTML(data, $pubTable, options);
            };
            $.get(bibSrc, callbackHandler, "text");
        }
```

**Comment by tdiethe** [comment]
Minor changes to TuningGuide's code:

`return function(bibSrc, bibElemId, opts) {
...
        var $bibSrc = null;
        if(bibsrc.indexOf('/') === -1) {
            $bibSrc = $(bibsrc);
        }

        if ($bibSrc && $bibSrc.length) { // we found an element, use its HTML as bibtex
            new Bib2HTML($bibSrc.html(), $pubTable, options);
            $bibSrc.hide();
        } else { // otherwise we assume it is a URL
            var callbackHandler = function(data) {
                new Bib2HTML(data, $pubTable, options);
            };
            $.get(bibsrc, callbackHandler, "text");
        }`

**Comment by vkaravir** [comment]
Thanks guys! It was an issue with newer jQueries than the ancient version used by the example. Should be fixed in commit e8c1c2ac410d2fb4a5e6656779bf09fba9d51efc along with other jQuery issues.
